js/testapps/json-mode/prompts/recipe.jsonmode.prompt

```diff
@@ -0,0 +1,12 @@
+---
+model: googleai/gemini-1.5-flash-latest
+input:
+  schema:
+    food: string
+output:
```

Comment:
Should we define the output schema here as opposed to in the code?

Reply:
For this prompt, there's no `output` schema defined either here or in the code; it is just specifying that the output is JSON mode. The LLM will (should!) return an adhoc JSON structure rather than a pre-defined one.